core/jazz_ui/src/app/pages/environment-overview/env-overview-section.component.ts

```diff
@@ -268,6 +439,12 @@ popup(state){
             this.isLoading = false;
           }
           else {
+            this.safeName = response.data.environment[0].metadata.safe.name
+            let ser = this.service.serviceType
```

Comment:
Please use more descriptive variable names. Also, please prefer semicolons at line endings or adhere to the dominant standard.

Reply:
done

Reply:
done